tsnet/tsnet.go

```diff
@@ -180,11 +182,19 @@ func (s *Server) start() error {
 
 	// TODO(bradfitz): start logtail? don't use filch, perhaps?
 	// only upload plumbed Logf?
+	pol := logpolicy.New(logtail.CollectionNode)
```

Comment:
I'm concerned about where this is going to locally store the logs data.

It appears to me that by default it figures it out from a bunch of environment variables:
https://github.com/tailscale/tailscale/blob/dce2409b15837f30885405b8b1d27e7b7fd6bf7a/logpolicy/logpolicy.go#L148-L218

However, I would have expect this to use `Server.Dir`.

Reply:
Part of the problem is that the `logpolicy` package was designed with the assumption that it's used within a main binary where only one instantiation ever occurs.

We may need to extend the API to support running many `logpolicy` instances within a single process.